README.md

```diff
@@ -1,101 +1,44 @@
-# lowRISC web site
+# lowRISC website
 
-This repository contains the source code of the lowRISC web site. It is built
-using Hugo, with all frontend assets being managed by Webpack.
+This repository contains the source code of the lowRISC website.
 
-## Develop
+## Deploying
 
-### Prerequisites
+Build the site for production with the following command:
 
-You need to install node.js and yarn on your machine. (Note: you do not need to
-install Hugo manually, it will be installed with yarn later.)
-
-- [nodejs](https://nodejs.org/en/download/)
-- [yarn](https://classic.yarnpkg.com/en/docs/install/)
-
-### Download all frontend dependencies
-
-Before you can get started, you need to download all frontend dependencies
-(CSS and JS libraries), and Hugo. These dependencies will be installed into the
-`node_modules` directory.
-
-```bash
-yarn install
+```sh
+hugo -s site -d dist --minify --baseURL 'https://www.lowrisc.org'
```

Comment:
We could extract this into a `build.sh` or a Makefile target or something.

Reply:
I've moved the Hugo directories into the top level, so the commands are now:

```sh
# Preview
hugo serve
# Preview with draft and future content
hugo serve -D -F
# Build for deployment
hugo --minify --baseURL 'https://www.lowrisc.org'
```

I agree the third target is worth a Makefile so I've added one, but at some stage if the deployment is done via GitHub CI I think the first two commands are simple enough for devs to run themselves.

Reply:
Just to clarify, the Makefile currently covers all three commands (`make`, `make preview`, `make preview-drafts`), but if `make` is eventually handled by CI automatically then I think `make preview` and `make preview-drafts` are simple enough not to need the Makefile any more.